### PR TITLE
Home blog

### PR DESCRIPTION
**Pull request by Vjerem**
Update HomeBlog

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
     -->
     <title>React App</title>
   </head>
-  <body>
+  <body style="background-color: #2d4059">
```
Comment (KeerhemFR) [inline public/index.html]: Has nothing to do with the HomeBlog, and changing an element style in the html elements besides by changing it in a css file is a bad practice

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.logo {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
```
Comment (KeerhemFR) [inline src/App.css]: ?

File: src/App.jsx [hunk 3]
```diff
@@ -1,5 +1,14 @@
...
+import './App.css';
+
 function App() {
-  return <h1>Hello there!</h1>;
+  return (
+    <div>
+      <img className="logo" src={wildHeroes} alt="wildHeroes" />
+      <HomeBlog />
+    </div>
+  );
```
Comment (KeerhemFR) [inline src/App.jsx]: Everything related to the HomeBlog should be load in the homepage.jsx beside the app.jsx

**Review by KeerhemFR — CHANGES_REQUESTED** [review]
Your part seems okay to me, but you worked on files which you shouldn't have, so I can't accept the PR
I left comments where it isn't good, please change that and the review will be okay for me.

**Review by KeerhemFR — DISMISSED** [review]
Allright, previous mistakes were corrected, everything is fine for me.

File: src/Components/HomeBlog/Index.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
+.HomeNews {
```
Comment (Hacene010) [inline src/Components/HomeBlog/Index.css]: use camelCase instead of PascalCase for the className.

File: src/Components/HomeBlog/Index.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  margin: auto auto;
+  padding-bottom: 10%;
+  background-color: #ffb400;
+  width: 75%;
+  border-radius: 10px;
+  text-align: center;
+}
+
+@media only screen and (min-width: 600px) {
+  .HomeNews {
```
Comment (Hacene010) [inline src/Components/HomeBlog/Index.css]: Same here

File: src/Components/HomeBlog/index.jsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import './Index.css';
+import ArticleHomeBlog from '../ArticleHomeBlog/index';
+import fight from './Images/fight.png';
+import cardsgame from './Images/cardsgame.png';
+import heroeslist from './Images/heroeslist.png';
+
+function HomeBlog() {
+  return (
+    <section className="HomeNews">
```
Comment (Hacene010) [inline src/Components/HomeBlog/index.jsx]: Same here (camelCase)

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some syntax errors to fix

**Review by Hacene010 — APPROVED** [review]
Ok but you need to fix the problem with your App.jsx

**Review by KeerhemFR — APPROVED** [review]
Everything seems fine to me

**Pull request merged**